doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -424,11 +416,34 @@ Response code:
 * Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
 
 + Parameters
-    + limit: 20 (optional, number)
-    + offset: 20 (optional, number)
-    + type: Room (optional, string)
-    + id: Bcn_Welt (optional, string)
-    + idPattern: (optional, string)
+    + id: Boe_Idearium (optional, string) - A comma separated list of elements.
+    Retrieve entities which ID match one of the elements in the list.
+    
+    + type: Room (optional, string) -  comma separated list of elements.
```

Comment:
that's something Fabre will show and I'm sure apiary will show in their standard rendering as well